content/docs/user-guide/gems/reference/aws/aws-gamelift/advanced-topics.md

```diff
@@ -0,0 +1,118 @@
+
+2. Look up the fleet with the ID `MyGameLiftFleetId` by using the **GetResourceNameId** node.
+
+3. Filter the search with a maximum of 10 sessions in the corresponding fleet by using the **FleetId** and **MaxResult** nodes.
+
+4. Search for sessions by using the **SearchSessionsAsync** node.
+   
+![A scipt that searches for a maximum of ten sessions in the corresponding fleet.](/images/user-guide/gems/reference/aws/aws-gamelift/searchactivesessions.PNG)
+
+
+### Example: Browse search results and join session
+
+After you get search results from the previous example, this script chooses the first game session and joins it. The script begins automatically after **SearchSessionsAsync** completes with results. 
+
+1. Set this script to begin automatically after searching for sessions by adding the **OnSearchSessionsAsyncComplete** event to **AWSGameLiftSessionAsyncRequestNotificationBus**, then applying that to the **SessionConfigs** node.
+   
+2. Get the first session from the results of the **SearchSessionsAsync** node by using the **Get First Element** and **SessionId** nodes.
+
+3. Get a request object to join the session by using the **Get joinrequest** and **SessionId** nodes.
+
+4. Create a player ID for the session by using the **SessionId** and **CreatePlayerId** nodes.
+
+5. Allow a player to join the session by using the **PlayerId** and **JoinSessionAsync** nodes.
+
+![A script that chooses a game from search results and joins the first session.](/images/user-guide/gems/reference/aws/aws-gamelift/searchandjoin.PNG)
+
+
+## Get credentials from an Amazon Cognito identity pool
+
+To make requests against GameLift, and to permit players to interact with your resources, you must get the players' AWS credentials. With the [AWS Client Auth Gem](/docs/user-guide/gems/reference/aws/aws-client-auth/), you can use players' login information to get their AWS credentials from an Amazon Cognito identity pool. To get players' credentials from an identity pool, they must successfully log in.
+
+After players log in, you can use the AWS GameLift Gem to configure the client and set up their AWS credentials by calling `AWSGameLiftClientManager::ConfigureGameLiftClient()`.
+
+
+## Set up log paths for multiple processes
+
+You can configure GameLift fleets to run multiple processes on each instance. This can help you use your hosting resources more efficiently and potentially help reduce overall hosting costs. For more information, refer to [Managing how game servers are launched for hosting](https://docs.aws.amazon.com/gamelift/latest/developerguide/fleets-multiprocess.html) in the Amazon GameLift Developer Guide.
+
+To launch a server process on a GameLift fleet and save the log files, enter the following command. You can set a unique log path for a server process by providing a unique identifier in the `project-path` launch parameter. In this example, log files are saved to `C:\game\process1\user\log\` instead of the default log path.
```

Comment:
"To support logging for multiple processes on the same GameLift instance, enter the following command."